src/database/schema.ts

```diff
@@ -0,0 +1,11 @@
+import {text, pgTable, serial} from "drizzle-orm/pg-core";
+
+export const users = pgTable ("users",
+    {
+        id: serial("id").primaryKey(),
```

Comment:
ident is a bit weird here. I suggest installing Prettier as a dev dependency and setup your IDE to format on save. This will make the code look uniform